agenta-cli/agenta/sdk/decorators/llm_entrypoint.py

```diff
@@ -72,16 +102,18 @@ class route(BaseDecorator):
     # the @entrypoint decorator, which has certain limitations. By using @route(), we can create new
     # routes without altering the main workflow entrypoint. This helps in modularizing the services
     # and provides flexibility in how we expose different functionalities as APIs.
-    def __init__(self, path):
-        path = "/" + path.strip("/").strip()
-        path = "" if path == "/" else path
-
+    def __init__(self, path, config_schema: BaseModel):
+        self.config_schema: BaseModel = config_schema
+        if path != "" and path[0] != "/":
```

Comment:
Be careful with this, you're changing the default value that you send to `@entrypoint`, which expects `""` as a default and not `"/"`